build.gradle

```diff
@@ -37,14 +38,12 @@ def dependencyVersions = [
         ida_utils: '326',
         ida_test_utils: '35',
         saml_extensions:"$opensaml-1.2a-42",
-        saml_serializers:"$opensaml-28",
-        saml_security:"$opensaml-58",
-        saml_metadata_bindings:"$opensaml-72",
-        saml_utils: "$opensaml-70",
-        saml_test_utils:"$opensaml-46",
-        saml_domain_objects: "$opensaml-44",
-        hub_saml: "$opensaml-115",
-        dev_pki: '1.1.0-19'
+        saml_serializers:"$saml_libs_version",
```

Comment:
I think we can just get rid of these individual version variables and use `$saml_libs_version` in their place everywhere.